src/arc.ts

```diff
@@ -421,6 +421,7 @@ async function getCurrentAccountFromProvider(): Promise<Address | null> {
  * Clear caches as every case where we're manually logging out
  * implies that the cache should be cleared
  */
+//eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed?  I suspect it is because you didn't figure out a better alternative to `any`. I think there likely _is_ a better way and would prefer that we find it.